csharp/OpenStudio.nuspec.in

```diff
@@ -5,19 +5,27 @@
     <version>@OpenStudio_VERSION@</version>
     <authors>NREL</authors>
     <owners>NREL</owners>
-    <licenseUrl>http://github.com/NREL/OpenStudio/blob/develop/LICENSE.md</licenseUrl>
+    <license type="file">LICENSE.md</license>
     <projectUrl>http://github.com/NREL/OpenStudio</projectUrl>
     <description>Whole building energy simulation</description>
     <tags>OpenStudio EnergyPlus Simulation Modeling</tags>
-    <references>
-        <reference file="OpenStudio.dll" />
-    </references>
+    <dependencies>
+      <group targetFramework=".NETFramework3.5" />
+    </dependencies>
```

Comment:
This isn't compatible with a nuget for dotnet, but whatever...

Reply:
Im ok if you want to make this a configured value, but generally speaking I made zero effort to make this work with dotnet core.

Reply:
dotnet core doesn't require "nuspec" file for nuget package.

Reply:
@MingboPend so full disclaimer: I suck at all things C#. I've tried in #3960 to scratch the nuspec file and include everything needed in .csproj, but I couldn't make it work with MSVC.